README.md

```diff
@@ -1,2 +1,143 @@
+
+Welcome to the “Blockchain India” Hackathon to change the world – for Social Good…
+
+### 1. Agriculture & Farming
+
+Making Supply Chain logistics farmer friendly and prevention of farmer suicides ; Linking farmers to markets in cities and international trade corridors like the Flower Auction Market in Bangalore 
+
+### 2. Freight & transportation 
+
+of goods and produce through blockchain enabled sharing-market reducing freight, cutting down fuel and de-congesting highwaysUsing 
+
+### 3. Artificial Intelligence & Blockchain 
+
+for weather-friendly crop advice to farmersTrade Finance for SMEsPeer to Peer loan market and on-line lending Person to Person Microinsurance for events like crop failure
+
+### 4. Blockchain for Governments
+
+Governance, Public Goods Subsidy & Administration 
+
+### 5. Blockchain for Finance 
+
+for Financial Inclusion & the unbanked
+
+### 6. Blockchain enabled Market mashups 
+like intersections of industry verticals 
+
+### 7. Blockchain for improving market efficiency 
+in capital markets, financial services, banks
+
+### 8. Supply chain 
+to track the origin of products: the lifecycle from raw material, through processing, assembly, logistics, retail and resale...
+
+### 9. Education , learning & certification 
+vocational training, life long learning, upskilling for the fourth industrial revolution 
+
+### 10. Counterfeiting of goods 
+like art, jewellery, gold, currency etc 
+
+### 11. Authentication of pharmaceuticals
+
+drugsHospitals, healthcare management, vaccinations, medical records & insuranceHospitality & hotel management, loyalty rewards
+
+### 12. Real estate management
+land registry, shared ownership 
+
+### 13. Shared economy
+room rentals like Airbnb,car rentals like uber, shared ownership of everything like gyms, golf clubs etc
+
+
+## Judging & Rewards
+
+There are four criteria for judging the submission in the Hackathon, they are:
+
+### Criteria
+
+- Innovative nature of the design – decentralization: 40%
+- Potential for Business Application: 20%
+- Quality of Execution and Pitch Impact: 20%
+- Social Impact: 20%
+
+Each judge will assign a score between 1 and 10 inclusively for each of the criteria with 1 being unexceptional or boring and 10 being exceptional and exciting.
+
+### A penalty system will apply.
+
+1. Using any technology / libraries that were not disclosed before judging as part of the Submission
+2. Checking any code into Github after the deadline. Judges will use their own discretion in deciding how much impact a penalty has on the team’s score
+3. A solution that does not have Blockchain at its core.
+
+### Outcomes
+
+The Hack the Future Hackathon is a problem-solving arena to incubate new ideas and find exceptional talent and match it with capital and reward.
+
+The Rewards <TODO: @arifa>
```

Comment:
5 shortlisted teams will be invited to Ethereum India Summit on 18 May at The Taj Mahal Palace Hotel to pitch to judges and Vitalik Buterin will make the final assessment and pick out the winner and hand out the cash prize of Rs 1 lakh to the winning team.